libs/sdk-core/src/models.rs

```diff
@@ -1169,32 +1169,53 @@ impl TryFrom<i32> for SwapStatus {
 /// saved to the persistent storage.
 ///
 /// The SwapInfo has a status which changes accordingly, documented in [SwapStatus].
+///
+/// Static fields are those which don't change for the lifetime of a swap, whereas dynamic fields may change.
+
 #[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
 pub struct SwapInfo {
-    //static immutable data
+    /// Bitcoin address for this swap. Sats sent to this address will be swapped. Static field.
```

Comment:
Static field doesn't seem the right term. Perhaps "immutable field" more fits here although I don't feel we should add that info.

Reply:
Good point, might be needless to add this info. Removed